source/java/spring-boot-web/index.html

```diff
@@ -7,7 +7,7 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   
-  <title>Spring Boot Stormpath Web Starter &mdash; Spring Boot Stormpath Web Starter 1.0.RC4.5 documentation</title>
+  <title>Spring Boot Stormpath Web Starter &mdash; Spring Boot Stormpath Web Starter 1.0.RC5-SNAPSHOT documentation</title>
```

Comment:
Yes. That's auto-generated. Once we release, I will rebuild the docs and update this PR.